python/doc/content/python/MooseDocs/extensions/appsyntax.md

```diff
@@ -28,7 +28,7 @@ description of the object. To display this text the `!syntax description` comman
 the MOOSE input file syntax for the object as in [appsyntax-description].
 
 !devel! example id=appsyntax-description caption=Example use of the `!syntax description` command.
-!syntax description /Mesh/FileMesh style=color:green
+!syntax description /Mesh/FileMesh style=color:green;
```

Comment:
This style parameter doesn't seem to be doing anything in the CI-rendered site. Are you seeing a difference locally?

Reply:
I agree, in the original file the text did not render green. I lazily did not try compiling with my change to see if it changed, but I'm guessing it didn't from your wording. The semicolon was my guess at fixing it -- since the coloring is not the purpose of this example, we could just remove the `style` part altogether.

Reply:
Yeah - I would assume it was a bit of flourish to show that it is possible to use `style`, but I imagine something about the CSS has changed since that example was made. Let's just remove that part.